src/main/java/pt/tqs/g205/resources/CategoriaPratoResource.java

```diff
@@ -22,11 +22,11 @@
   private CategoriaPratoService categoriaService;
   
   /**
-   * Endpoint para conseguir todos os pratos.
-   * @return JSON com descricao de todos os pratos.
+   * Obter todos os pratos.
+   * @return lista de todos os pratos.
    */
   @RequestMapping(value = "/pratos", method = RequestMethod.GET)
-  public ResponseEntity<List<CategoriaPrato>> getAll() {
+  public ResponseEntity<?> getAll() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/bdomingues-github/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=pt.tqs.g205%3Arestapi&pullRequest=6&issues=AWNEkG2UXOBV_Yrh_0kc&open=AWNEkG2UXOBV_Yrh_0kc)

[ISSUE_KEY]: AWNEkG2UXOBV_Yrh_0kc